k8s/bases/core-cc/core-cc-task-manager-configmap.yaml

```diff
@@ -85,12 +85,13 @@ data:
           - CGM_OUT
           - METADATA
         enable-export-logs: false
+        manual-upload-base-path: CORE/CC
       whitelist:
         - http://gridcapa-minio:9000/
 
     minio-adapter:
       bucket: gridcapa
-      base-path: ${MINIO_BASE_DIRECTORY}
```

Comment:
Did you forget to remove the `MINIO_BASE_DIRECTORY` variable from the `core-cc-task-manager-deployment`?

Reply:
no in fact that is how it was set beforer .... MINIO_BASE_DIRECTORY isn't set in docker local configs